docs/site/migration/auth/built-in.md

```diff
@@ -1,17 +1,290 @@
-" %}
+This is a guide to describe how to migrate LoopBack 3 built-in User/AccessToken
+model based authentication.
 
-{% include note.html content="
-This is a placeholder page, the task of adding content is tracked by the
-following GitHub issue:
-[loopback-next#3719](https://github.com/strongloop/loopback-next/issues/3719)
-" %}
+As the pre-work of this guide, we created an application completely migrated
+from
+[`loopback-example-access-control`](https://github.com/strongloop/loopback-example-access-control).
+It builds with a LoopBack 4 authentication system based on
+`@loopback/authentication` and an authorization system based on
+`@loopback/authorization`. Our migration guide will use it as the reference. A
+token based authentication module(prototype) is also exported as a component in
+the example for users to leverage.
+
+## LoopBack 3 Authentication
+
+You can take a few minutes on reading
+[this page](https://loopback.io/doc/en/lb3/Introduction-to-User-model-authentication.html)
+to recall the authentication system in LoopBack 3.
+
+LoopBack 3 provides a built-in `User` model, which can be easily migrated to
+LoopBack 4 by `lb4 import-lb3-models`. It also provides an `AccessToken` model
+to store a logged in user's information. Its authentication system's mechanism
+is:
+
+When a user logs in, the system:
+
+- creates a new access token
+- the token includes user info as attributes
+- stores the token in a database
+- returns token's id
+
+After setting the token in the explorer, subsequent requests will include the
+token in their headers, the system:
+
+- finds token record by id
+- returns the user info in the record
+- passes user info to other modules in the system
+
+Here is the diagram that describes the system:
+
+![LoopBack 3 authentication](../../imgs/tutorials/authentication-migration/lb3-authentication-user.png)
+
+## LoopBack 4 Authentication
+
+We highly recommend you read and understand the document of LoopBack 4
+authentication system before the migration:
+
+- [Introducing component `@loopback/authentication`](https://loopback.io/doc/en/lb4/Loopback-component-authentication.html)
+- [Tutorial of how to secure your LoopBack 4 APIs with jwt strategy](https://loopback.io/doc/en/lb4/Authentication-Tutorial.html)
+
+As a summary for the above documents: LoopBack 4 has a more flexible
+authentication module that can potentially support any authentication
+strategies, not only the token based one. To get an equivalent authentication
+system in your LoopBack 4 application, you can achieve that easily and fast by
+migrating the LoopBack 3 `User` model and plugging in a prototype JWT
+authentication component to perform the user login, and to verify a user's
+identity by token.
+
+Here is a diagram that illustrates the scenario in LoopBack 4, boxes with a
```

Comment:
`..in LoopBack 4. And the dark blue boxes are where it is different from LoopBack 3 authentication system`? 

( I am not sure about this, feel free to ignore)